classes/label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_reader :id, :title, :color, :items
+
+  def initialize(title, color)
+    @id = Random.rand(1..100)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    items.push(item)
+    item.add_label(self)
+  end
```

Comment:
![image](https://user-images.githubusercontent.com/99597333/201999796-c3fd5cf7-3aea-4041-b8a6-89fe9ff7c1ec.png)
if You're trying to implement the above requirement 
then you don't really to define another function inside the item_class
```
def add_item(item)
   items.push(item)
   item.label = self 
end

```
try the above solution and see if it works for you